ablator/modules/storage/remote.py

```diff
@@ -73,6 +142,20 @@ def rsync_up(
         timeout_s: int | None = None,
         run_async=False,
     ):
+        """
+        Upload a file to the remote storage.
```

Comment:
start a new process and upload

Reply:
Do you think "Asynchronous upload of a file to remote storage" better or not?

Reply:
I think it's not just a file, it's syncing from local folder to destination folder, aka upload all files in local folder to destination